nix/default.nix

```diff
@@ -70,6 +70,35 @@ let
           + shellHook;
       };
 
+  buildPursDocsSearch =
+    { name ? "purescript-docs-search"
+    , ...
+    }:
+    pkgs.stdenv.mkDerivation {
+      inherit name;
+      srcs = [
+        (pkgs.fetchurl {
```

Comment:
I believe it's possible to wrap the [NPM distribution ](https://www.npmjs.com/package/purescript-docs-search) as a normal node package which provides a binary.